### PR TITLE
Clarify the policy template documentation

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

**Comment by dockerymick** [comment]
@mprahl Thank you for creating the issue and starting the PR, Matt. I have made a few updates. When you have an opportunity, will you rereview the PR. We are making great progress with our issues, so 2.6 changes will be in shortly

File: governance/custom_template.adoc [hunk 1]
```diff
@@ -45,7 +45,11 @@ The `fromSecret` function returns the value of the given data key in the secret.
 func fromSecret (ns string, secretName string, datakey string) (dataValue string, err error)
 ----
 
-When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You receive a policy violation if the Kubernetes `Secret` resource does not exist on the target cluster. If the data key does not exist on the target cluster, the value becomes an empty string. View the following configuration policy that enforces a `Secret` resource on the target cluster. The value for the `PASSWORD` data key is a template that references the secret on the target cluster:
+When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You must use the same namespace that is used for the policy within the hub cluster template. See, <<hub-templates,Support for hub cluster templates in configuration policies>> for more details.
```
Comment (mprahl) [inline governance/custom_template.adoc]:
```suggestion
When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You must use the same namespace that is used for the policy when using the function in a hub cluster template. See, <<hub-templates,Support for hub cluster templates in configuration policies>> for more details.
```

File: governance/custom_template.adoc [hunk 2]
```diff
@@ -85,7 +89,7 @@ The `fromConfigmap` function returns the value of the given data key in the Conf
...
+When you use this function, enter the namespace, name, and data key of a Kubernetes `ConfigMap` resource. You must use the same namespace that is used for the policy within the hub cluster template. See, <<hub-templates,Support for hub cluster templates in configuration policies>> for more details. You receive a policy violation if the Kubernetes `ConfigMap` resource does not exist on the target cluster. If the data key does not exist on the target cluster, the value becomes an empty string. View the following configuration policy that enforces a Kubernetes resource on the target managed cluster. The value for the `log-file` data key is a template that retrieves the value of the `log-file` from the ConfigMap, `logs-config` from the `default` namespace, and the `log-level` is set to the data key `log-level`.
```
Comment (mprahl) [inline governance/custom_template.adoc]:
```suggestion
When you use this function, enter the namespace, name, and data key of a Kubernetes `ConfigMap` resource. You must use the same namespace that is used for the policy using the function in a hub cluster template. See, <<hub-templates,Support for hub cluster templates in configuration policies>> for more details. You receive a policy violation if the Kubernetes `ConfigMap` resource does not exist on the target cluster. If the data key does not exist on the target cluster, the value becomes an empty string. View the following configuration policy that enforces a Kubernetes resource on the target managed cluster. The value for the `log-file` data key is a template that retrieves the value of the `log-file` from the ConfigMap, `logs-config` from the `default` namespace, and the `log-level` is set to the data key `log-level`.
```

**Comment by dockerymick** [comment]
@mprahl thanks for rereviewing. I accepted your suggestions and also verified that the reference links to the section works. Looks good to merge.

**Comment by mprahl** [comment]
Thanks @dockerymick!

**Pull request merged**